src/authentication.ts

```diff
@@ -41,6 +42,37 @@ export async function redirectToLogin(config: TInternalConfig, customState?: str
   })
 }
 
+export async function redirectToRegister(config: TInternalConfig, customState?: string): Promise<void> {
+  // Create and store a random string in sessionStorage or localStorage, used as the 'code_verifier'
+  const storage = config.storage === 'session' ? sessionStorage : localStorage
```

Comment:
Same here, I guess.

But are there any cases where the register-page flow will allow sessionStorage to be sufficient, or does it always require another tab/window?